DrawerKit/DrawerKit/Internal API/PresentationController+Utilities.swift

```diff
@@ -18,18 +18,3 @@ func smallerThanOrEqual(_ lhs: CGFloat, _ rhs: CGFloat) -> Bool {
 func greaterThanOrEqual(_ lhs: CGFloat, _ rhs: CGFloat) -> Bool {
     return lhs > rhs || equal(lhs, rhs)
 }
-
-extension DrawerState {
-    public static func ==(lhs: DrawerState, rhs: DrawerState) -> Bool {
-        switch (lhs, rhs) {
-        case (.dismissed, .dismissed),
-             (.partiallyExpanded, .partiallyExpanded),
-             (.fullyExpanded, .fullyExpanded):
-            return true
-        case let (.transitioning(lhsCurPosY), .transitioning(rhsCurPosY)):
-            return equal(lhsCurPosY, rhsCurPosY)
```

Comment:
Removing this implementation is **not** the correct thing to do. The compiler-generated conformance will simply use `lhsCurPosY == rhsCurPosY` but these are floating point values that need to be compared, within a tolerance. There's a reason why `equal()` was implemented and used here. When the user drags the drawer and happens to let go a wee bit before or after the Y value associated with a state, the `==` equality will consider the Y values different when they should be considered equal. I ran into bugs when I first implemented DK using `==`, which is why I had to use `equal()`.